src/net/server.rs

```diff
@@ -87,6 +89,33 @@ impl HttpPeer {
         }
     }
 
+    // This function connects to the burndb, peerdb, and stacks chainstate dbs
+    //   _it does not call connect()_
+    pub fn spawn(mut self, my_addr: &SocketAddr, mut peerdb: PeerDB,
```

Comment:
Please decorate this with `#[cfg(test)]`.

Reply:
Moved into testnet::helium::node in 424419626